src/solver/solvers/solver_ppc.h

```diff
@@ -132,151 +182,335 @@ namespace solver
 
 
     template<typename HybridModel, typename AMR_Types>
-    void SolverPPC<HybridModel, AMR_Types>::advanceLevel(
-        std::shared_ptr<SAMRAI::hier::PatchHierarchy> const& hierarchy, int const levelNumber,
-        IPhysicalModel<AMR_Types>& model,
-        amr::IMessenger<IPhysicalModel<AMR_Types>>& fromCoarserMessenger, const double currentTime,
-        const double newTime)
+    void
+    SolverPPC<HybridModel, AMR_Types>::advanceLevel(std::shared_ptr<hierarchy_t> const& hierarchy,
+                                                    int const levelNumber, IPhysicalModel_t& model,
+                                                    IMessenger& fromCoarserMessenger,
+                                                    const double currentTime, const double newTime)
     {
         // bool constexpr withTemporal{true};
```

Comment:
still needed?

Reply:
don't even know what that was... removed